packages/features/ee/workflows/components/EventWorkflowsTab.tsx

```diff
@@ -223,13 +232,17 @@ function EventWorkflowsTab(props: Props) {
       });
       const disabledWorkflows = data.workflows.filter(
         (workflow) =>
+          (!workflow.teamId || eventType.teamId === workflow.teamId) &&
           !workflows
             .map((workflow) => {
               return workflow.id;
             })
             .includes(workflow.id)
       );
-      setSortedWorkflows(activeWorkflows.concat(disabledWorkflows));
+      const allSortedWorkflows = workflowsDisableProps.isLocked
```

Comment:
for the parent managed event type we want to list all team workflows here even when locked